### DISCUSSION TITLE
Sign ethereum transaction with amount less than 1

### DISCUSSION THREAD

**Discussion by chamid** [trustwallet/wallet-core#2062] [Q&A] [answered]
Hi
I have a headache to sign and ethereum transaction with value less than 1, eg 0.002
I saw tests and example and all of them are BigInntegers or in hex, and not a float number
Can please any one help me???

**Comment by optout21** [comment, marked as answer]
Ethereum uses 18 decimals, so 2000000000000000000 is 2 ETH.
See e.g.:
https://github.com/trustwallet/wallet-core/blob/master/tests/Ethereum/TWAnySignerTests.cpp#L162
